.travis.yml

```diff
@@ -77,6 +77,10 @@ before_install:
   - pushd builds
   # Build into own virtualenv
   # We therefore control our own environment, avoid travis' numpy
+  #
+  # Some change in virtualenv 14.0.5 caused `test_f2py` to fail. So, we have
+  # pinned `virtualenv` to the last known working version to avoid this failure.
```

Comment:
Yeah, I thought I saw some sort of Travis CLI tool at some point to, but the docs on this seem to disappear, as well.